packages/e2eTesting/cypress.json

```diff
@@ -1 +1,15 @@
-{}
+{
+  "integrationFolder": "integration",
+  "testFiles": "**/*.spec.ts",
+  "fixturesFolder": "fixtures",
+  "pluginsFile": "plugins/index.ts",
+  "supportFile": "support/index.ts",
+  "videosFolder": "videos",
+  "screenshotsFolder": "screenshots",
+  "reporter": "junit",
+  "reporterOptions": {
+    "mochaFile": "reports/junit-[hash].xml"
+  },
+  "viewportWidth": 1366,
+  "viewportHeight": 768
```

Comment:
Do we really want a viewport that small?

Reply:
I just copied what commerce use in the admin, it looks like a laptop resolution though. I don't mind changing it in the future if Gab or PO have any problem with it